packages/maskbook/src/plugins/ITO/hooks/usePoolPayload.ts

```diff
@@ -21,6 +21,6 @@ async function suspender(pid: string) {
     try {
         storage.set(pid, await PluginITO_RPC.getPool(pid))
     } catch (error) {
-        throw error
+        storage.set(pid, {} as JSON_PayloadInMask)
```

Comment:
This will cause an error in `<ITO />` component which will be captured by its wrapper `<ITO_LoadingFail />`

Reply:
what if you throw error directly?

Reply:
It will cause a dead loop which keep throwing error, and the UI is keeping ITO loading.